desktop/src/main/java/bisq/desktop/main/funds/transactions/TransactionsView.java

```diff
@@ -218,15 +208,15 @@ public void onKeysAdded(List<ECKey> keys) {
         };
 
         keyEventEventHandler = event -> {
+            // Not intended to be public to users as the feature is not well tested
             if (Utilities.isAltOrCtrlPressed(KeyCode.R, event)) {
                 if (revertTxColumn.isVisible()) {
                     confidenceColumn.getStyleClass().remove("last-column");
                 } else {
                     confidenceColumn.getStyleClass().add("last-column");
                 }
                 revertTxColumn.setVisible(!revertTxColumn.isVisible());
-            } else if (Utilities.isAltOrCtrlPressed(KeyCode.A, event))
```

Comment:
Is there an issue tracking the effort to update the website with the new fee information? For example, the Fees section of https://docs.bisq.network/exchange/whitepaper.html still lists that part of the fee goes to the arbitrator.

Reply:
Oh the doc is not updated then. @m52go Could you have a look?

BTC fees go to the Bisq Donation address defined in the DAO (can be changed by voting). From that the owner of that address will buy BSQ with the BTC and burn the BSQ, so converting the fee back into the BSQ ecosystem.

Reply:
That doc is obsolete [and archived](https://docs.bisq.network/archive.html).

See "Note on BTC revenues" [here](https://docs.bisq.network/user-dao-intro.html#traders-consume-burning-bsq).

Reply:
Maybe we should <noindex> it then? It is the second search result, and featured snippet, for "bisq fees" and when I was trying to understand the system more it was the first place I went to. I now see that the text at the top says it is archived, but with the search results jumping directly to sections it may cause confusion for new users. Just my 2c.

Reply:
Good idea, will do...I had no idea that document was ranking so high in search results.